File: CarDealership/src/main/java/rs/ac/uns/ftn/carDealership/model/carDealership/CarDealership.java
package rs.ac.uns.ftn.carDealership.model.carDealership;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.hibernate.annotations.GenericGenerator;

import javax.persistence.*;
import java.util.UUID;

@Getter
@Setter
@NoArgsConstructor
@Entity
@Table(name = "car_dealerships")
public class CarDealership {
    @Id
    @GeneratedValue(generator = "uuid2")
    @GenericGenerator(name = "uuid2", strategy = "uuid2")
    @Column(name = "car_dealership_id", nullable = false, unique = true)
    private UUID carDealershipId;

    @Column(name = "name")
    private String name;

    @Column(name = "address")
    private String address;

    @Column(name = "email")
    private String email;

    @Column(name = "phone_number")
    private String phoneNumber;

}

File: car-dealership-frontend/src/app/model/CarDealership.ts
export class CarDealership {
  carDealershipId: string;
  name: string;
  address: string;
  email: string;
  phoneNumber: string;
  workTime: WorkTime;
  constructor() {
    this.carDealershipId = '';
    this.name = '';
    this.address = '';
    this.email = '';
    this.phoneNumber = '';
    this.workTime = new WorkTime();
  }
}

export class WorkTime {
  day: string;
  startTime: string;
  endTime: string;
  constructor() {
    this.day = '';
    this.startTime = '';
    this.endTime = '';
  }
}
